app/(api)/_datalib/auth/verifyLogin.ts

```diff
@@ -0,0 +1,40 @@
+import bcrypt from 'bcrypt';
```

Comment:
npm i @types/bcryptjs

maybe use this as a dev dependency to fix up this error.